common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -398,7 +398,7 @@ trait PrebidSwitches {
     owners = group(Commercial),
     safeState = Off,
     sellByDate = never,
-    exposeClientSide = false
+    exposeClientSide = true
```

Comment:
Perhaps add a comment to explain why this needs to be exposed client-side.  It won't be obvious for someone looking at the code.

Reply:
Will do.

> Perhaps add a comment to explain why this needs to be exposed client-side. It won't be obvious for someone looking at the code.